src/components/Navbar/MobileNav.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { IconBrandGithub } from '@tabler/icons-react';
+import { useLocation } from 'react-router-dom';
+import { Burger } from '@mantine/core';
+import logo from '../../../public/logo.svg';
+import { Button } from '../UI/Button/Button';
+import { Divider } from '../UI/Divider/Divider';
+import { Link } from '../UI/Link/Link';
+import { Typography } from '../UI/Typography/Typography';
+import { NAVIGATION_LINKS } from './Navbar';
+import { mobileStyles as styles } from './styles';
+
+export default function MobileNav() {
+  const [displayLinks, setDisplayLinks] = useState(false);
+  const location = useLocation();
+  const pathname = location.pathname;
+
+  function handleDisplayLinks() {
+    setDisplayLinks((prev) => !prev);
+  }
+
+  return (
+    <div>
+      <div style={styles.container}>
+        <div style={styles.headerContainer}>
+          <img alt='Deadlink logo' style={styles.logoSize} src={logo} />
+          <Link labelStyle={styles.header} href='/' label='Deadlink-Hunter' disableHover />
+        </div>
+        <Burger
+          style={styles.burger}
+          size={styles.burgerSize}

```

Comment:
2 questions: 
1) cant you change the width and height of the Burger from `styles.burger` ? 
2) Why do we need a style object for the size? the size is a number we can just pass it